tests/S-10_Edit_Class.test.js

```diff
@@ -20,69 +20,90 @@ test.after.always(async (t) => {
     console.log("Testing finished, closing test server")
     await t.context.server.close();
     console.log("Successfully closed test server")
-
 })
 
+/**
+ * Class object to be used in tests
+ */
+const exampleClassObject = {
+    className: "maths",
+    users: [
+        {
+            grade: 6.027456183070403,
+            user: {
+                surname: "surname",
+                name: "name",
+                id: 0,
+                userName: "userName",
+                email: "email",
+            },
+        },
+        {
+            grade: 6.027456183070403,
+            user: {
+                surname: "surname",
+                name: "name",
+                id: 1,
+                userName: "userName",
+                email: "email",
+            },
+        },
+    ],
+};
+
 // ------------------ Test putClassInfoAdmin & PUT /admin/class/{className} ------------------
 
 // Test service function
 test("putClassInfoAdmin", async (t) => {
-    // The swaggerhub-generated backend doesn't do anything with the request
-    // body, so we cannot truly test it. We may leave it empty for now.
-    const requestBody = {
-        json: {}
-    }
+    const classroom = exampleClassObject
+    // The swaggerhub-generated backend doesn't do anything, so we can only
+    // test that it doesn't throw
```

Comment:
Doesn't throw what? 
Something is _missing_ 
Perhaps we should make it more precise and change it to "test that it doesn't throw an error"